.github/workflows/build.yaml

```
@@ -0,0 +1,22 @@
+
+name: Main Workflow
+jobs:
+  sonarqube:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v4
+      with:
+        fetch-depth: 0
+    - name: SonarQube Scan
+      uses: sonarsource/sonarqube-scan-action@v3.0.0 
+        SONAR_TOKEN: ${{ secrets.SONAR_TOKEN }}
```

actionlint

could not parse as YAML: yaml: line 21: mapping values are not allowed in this context

(syntax-check)